docs/checkit.js

```diff
@@ -354,12 +356,92 @@ CheckIt.prototype.onAuthStateChanged = function(user) {
     }
 };
 
-// Returns true if user is signed-in. Otherwise false and displays a message.
-CheckIt.prototype.checkSignedInWithMessage = function() {
-    // Return true if the user is signed in Firebase
-    if (this.auth.currentUser) {
-        return true;
-    }
+CheckIt.prototype.generateEmptyCalendar = function(calObj) {
+    // Generate the html for an empty calendar of the calendar you want to 
+    // display.
+    
+    var calendarDiv = this.$calendarDiv;
+    
+    // Add the title of the calendar
+    calendarDiv.append('<div id="calendarTitleHeading"> <h1 class="page-header text-center">' +
+                        calObj.state.title + '</h1></div>');
+                        
+
+    calObj.monthObjects.forEach (function(monthObj, index) {
+
+        var yearHeader = "<div class='page-header text-center'>" +
+                         "<h2 id='yearHeader'>" + monthObj.monthYear + "</h2>" +
+                         "</div>";
+        
+        //the div ID is the monthID
+            
+        calendarDiv.append('<div class="monthframe" id=' + monthObj.monthId + '></div>');
+        
+        if (self.monthIndex === 0) {
+            $('#' + monthObj.monthId).append(yearHeader);
+        }
+        $('#' + monthObj.monthId).append($('#template').html());
+            
+    });
+    
+};
+
+CheckIt.prototype.fillCalendar = function(calObj) {
+    // Fill an empty calendar with appropriate calendar data.
+
+    calObj.monthObjects.forEach (function(monthObj) {
+        
+        var $monthId = $('#'+ monthObj.monthId);
+        
+        $monthId.find(".month-year").empty();
+        $monthId.find(".month-year").append(monthObj.monthName + " " + monthObj.monthYear);
```

Comment:
Are these two lines together are the same as `$monthId.find(".month-year").text(...)`?